package.json

```diff
@@ -17,21 +18,25 @@
     "@ethersproject/contracts": "^5.0.1",
```

Comment:
`@babel/runtime` shouldn't be needed as the Babel config doesn't use the runtime plugin